.Gruntfile.babel.js

```diff
@@ -333,7 +337,7 @@ module.exports = (grunt) => {
         // scss({ output: `${distCSS}/scss.css` }), // FAIL - produces empty bundle, probably only
         //                                              useful in the <script> section, i.e.
         //                                              <script>import 'foo.scss' ...
-        eslint({ throwOnError: true, throwOnWarning: true }),
+        eslint({ throwOnError: true, throwOnWarning: true, fix: true }),
```

Comment:
@pieer @taoeffect and @SebinSong 
While discovering how "watch" works with rollup and eslint I discovered this little big tip!

Now, when we face a new eslint error during development, (`grunt dev`) it automatically tries to fix it when possible and keeps running the process as usual.

For example, if you write `const foo = 1;`, usually, it would throw the error about the extra semicolon and it wouldn't reload the page. Now, it automatically removes the semicolon and you don't need to manually remove (fix) it.

This saves us precious seconds each time we miss-write something, which is a tone of time at the end of day coding around ⚡️ 

Note that this doesn't run on build, only during watch mode.

Reply:
Sorry @sandrina-p, I think I've mentioned this before, this kind of auto-fixing stuff is extremely dangerous to add. It is never OK for the compiler or linter to write code for you without your explicit permission. It should always be a conscious step.

Reply:
Initially, when I mentioned this a year ago it was to run all the time, now it's only during development. But I understand your concern. Removed.

I hope it helps those who find it useful.